src/models/products.ts

```diff
@@ -2,6 +2,7 @@ import { Model, DataTypes, CreationOptional } from 'sequelize';
 import { connection } from '../database';
 import Category from './categories';
 import ExtraItem from './extraItems';
+import Sale from './sales';
```

Comment:
Same import issue here, lets declare the association in the associate method and that we we can avoid this import